### ISSUE TITLE
In-line comments

### ISSUE THREAD

**Issue by jnovembre** [jhmarcus/ggv_paper#2]
I put several comments in-line in the paper (forgetting how awesome github issue tracking is!). 

They are findable by searching on 'xx'...

**Comment by jhmarcus** [comment]
Awesome! Thanks for the comments! I'll take another stab at it asap.

**Comment by jhmarcus** [comment]
Made a solid push on things:
1. Reformatted the manuscript in latex as I was getting some formatting issues in using the Rmd 
2. Converted all the figures to pdf which is looking a lot better (except for the screen shot of the ggv website). Found a nice tool called `rsvg-convert` for converting svg to pdf. 
3. Simplified Figure 1
4. Fixed up most of your edits (I'd still like to discuss the best approach for figure 4. I started to replicate the table you showed me as a latex table (at the end of the pdf)
5. Moved the rare variant section to the conclusion / added some future directions

There are still a couple more references to fill in but its getting closer!  Note I've been using rstudio for compiling the tex / bib files.
